src/static/riot/profiles/user_detail.tag

```diff
@@ -0,0 +1,177 @@
+<user-detail>
+    <h1>User Edit Screen</h1>
+    <form class="ui form" id="user-form">
+        <div class="field">
+            <label>Profile Photo</label>
+            <label show="{ photo }">
+                Uploaded Photo: <a href="{ photo }" target="_blank">{ photo_name }</a>
+            </label>
+            <div class="ui left action file input">
+                <button class="ui icon button" onclick="document.getElementById('profile_phtoto').click()">
+                    <i class="attach icon"></i>
+                </button>
+                <input id="profile_phtoto" type="file" ref="photo" accept="image/*">
```

Comment:
If I click the profile field, it drops a file picker, as it should.

If I click any other field and then press enter to submit when I'm done, it submits the form and, now unexpectedly, displays a file picker even after redirecting to the profile detail page. Sort of a nitpick, but kind of annoying.
![image](https://user-images.githubusercontent.com/21694127/99608005-e60d7280-29c1-11eb-9666-ab9588dec0f0.png)

Reply:
Fixed